src/Comment/Comment.js

```diff
@@ -8,17 +8,21 @@ export const Comment = (props) => {
       role="comments"
       type="comments"
       className="comments">
+
       <div className="user-container">
         <h1 className="user">{props.comment.author}</h1> 
         <h1 className="time">3 months ago</h1> 
       </div>
+
      <p className="comment-text">{props.comment.comment}</p>
+     
      <div className="buttons-container">
        <i className="far fa-thumbs-up"></i>
-       <h3 className="like-count">1</h3>
+       <h3 className="like-count">0</h3>
```

Comment:
Is this for the ability for people to like each other's comments?

Reply:
that is correct 👍